src/storage/v2/edge_accessor.cpp

```diff
@@ -264,6 +265,29 @@ Result<PropertyValue> EdgeAccessor::GetProperty(PropertyId property, View view)
   return std::move(value);
 }
 
+Result<uint64_t> EdgeAccessor::GetPropertySize(PropertyId property, View view) const {
+  if (!storage_->config_.items.properties_on_edges) return 0;
+
+  {
+    auto guard = std::shared_lock{edge_.ptr->lock};
+    Delta *delta = edge_.ptr->delta;
+    if (!delta) {
+      return edge_.ptr->properties.PropertySize(property);
+    }
+  }
+
+  auto property_result = this->GetProperty(property, view);
+
+  if (property_result.HasError()) {
+    return property_result.GetError();
+  }
+
+  auto property_store = storage::PropertyStore();
+  property_store.SetProperty(property, *property_result);
```

Comment:
You have not applied deltas in this case, the property that was got, maybe inconsistent with isolation mode. Same applies for VertexAccessor